INSTALLATION.md

```diff
@@ -70,6 +60,27 @@ Congratulations! You have successfully installed **MassaStation** on your comput
 
 Congratulations! You have successfully installed **MassaStation** on your computer. You can start **MassaStation** by searching for it in the Applications folder.
 
+### GUI Installation
+
+1. Download the latest version of **MassaStation** installer for Debian Linux (`.deb`) from [here](https://github.com/massalabs/station/releases/latest/).
+2. Open your file manager and navigate to the location where the `.deb` package is saved.
+3. Right-click on the `.deb` package and choose "Open with Software Install" or a similar option.
+4. The package manager will launch and display **MassaStation** installation page.
+5. Review the package information and dependencies, if any, and click on the "Install" button.
+6. If prompted, enter your administrator password to authorize the installation.
+7. The installation will commence, and you will see a progress bar indicating the status.
+8. Once the installation is complete, you will receive a notification confirming the successful installation.
+
+#### GUI update:
+
+1. Before proceeding to the update of Massastation, run the following command to uninstall the package:
+
+```bash
+sudo dpkg -r massastation
```

Comment:
should we say to uninstall Massa Station from the Software Install program?

Reply:
I'll check the flow and make sure it works correctly

Reply:
Flow is working i'll update rest of document additions